src/Menu/index.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import './style.scss';
+import menuDivider from '../image/menu-divider.png';
+import imageLine from '../image/line.png';
+
+
+function Menu() {
+  return (
```

Comment:
too long component with the same divs. Can we move data to Array and use map function for example and render it?

Reply:
changed to function in another file